README.md

```diff
@@ -10,6 +10,20 @@ This project is designed to track progress within #100DaysOfCode.
 
 The data for this application is stored in LocalStorage. It can be saved and loaded as a JSON file.
 
+## Run with Docker
```

Comment:
Can we move the `## Run with Docker` into the `## Project Development Notes` (as a subset `###`)?

Reply:
I moved the part to the bottom of the project development. heading.